course/constants.py

```diff
@@ -368,6 +368,8 @@ class flow_permission:  # noqa
     set_roll_over_expiration_mode = "set_roll_over_expiration_mode"
     see_session_time = "see_session_time"
     lock_down_as_exam_session = "lock_down_as_exam_session"
+    send_interact_email_to_course_staff =\
+        "send_interact_email_to_course_staff"
```

Comment:
Not sure I get why this needs to be per-page, rather than one of the (new) general `participation_permission`s?